scripts/sdk/build.sh

```diff
@@ -12,19 +16,10 @@ mkdir target/paths
 # this doesn't work, the ?url imports need to be in the consuming project, so disable for now
 # ./scripts/sdk/transform-paths.js ./src/platform/web/sdk/paths/vite.js ./target/paths/vite.js
 cp doc/SDK.md target/README.md
-pushd target
-pushd asset-build/assets
-mv main.*.js ../../main.js
-# Create a copy of light theme for backwards compatibility 
-cp theme-element-light.*.css ../../style.css
-# Remove asset hash from css files
-mv theme-element-light.*.css ../../theme-element-light.css
-mv theme-element-dark.*.css ../../theme-element-dark.css
-mv download-sandbox.*.html ../../download-sandbox.html
-rm *.js *.wasm
-mv ./* ../../
+pushd target/asset-build
+rm index.html
```

Comment:
> [...] when running `yarn build:sdk` or `yarn watch:sdk`, I get this error:
>
> ```
> [...]
> rm: cannot remove 'index.html': No such file or directory
> ```
>
> *-- https://github.com/vector-im/hydrogen-web/pull/693#pullrequestreview-973791417*

Fixed this up by removing `index.html` in the right spot (`target/asset-build/index.html`).

Sorry didn't pick up on this earlier. I think it's because when running on Windows the git bash window which runs when you `yarn build:sdk` closes so fast and masks errors for me. I've tested this on macOS and everything is working now 👍